platforms/core-runtime/java-language-extensions/src/main/java/org/gradle/api/JavaVersion.java

```diff
@@ -326,6 +326,15 @@ public String getMajorVersion() {
         return String.valueOf(ordinal() + 1);
```

Comment:
💅 should this line also use `getMajorVersionNumber()`?

Reply:
Absolutely

Reply:
Fixed!